models/staging/ga4/stg_ga4__events.sql

```diff
@@ -61,7 +61,18 @@ enrich_params as (
         {{extract_hostname_from_url('page_location')}} as page_hostname,
         {{extract_query_string_from_url('page_location')}} as page_query_string,
     from remove_query_params
+),
+fix_params as (
+    select
+        * except (medium, campaign),
+        case
+            when page_query_string like '%gclid%' then "cpc"
```

Comment:
I think I'm missing what the difference is between these 2 case statements. Aren't they both searching for the same thing? `page_query_string like '%gclid%'